docs/manual/Quick-Start Guides/11-traefik-on-port80.md

```diff
@@ -3,3 +3,5 @@
 #### Video Guide
 
 ![type:video](https://www.youtube.com/embed/UHuyn9qCY6g)
+
+Afterwards you can follow `16 - Setting up External-Services` if you don't want to access the TrueNAS web gui using the new ports (port 81 and 444 in the video guide). When setting up the `External Service`, set `External Service IP` to the ip address of your TrueNAS server, `Port Type` to `HTTP` and `Service Port` to `81`. You can set the `Host` and `HostName` values to the hostname of your TrueNAS server e.g. truenas.example.com. Now Traefik will serve the TrueNAS web gui over HTTPS on truenas.example.com. Ensure your DNS points truenas.example.com to the ip address of your TrueNAS server.
```

Comment:
Also some actual feedback, besides wanting this in another file:

> When setting up the `External Service`, set `External Service IP` to the ip address of your TrueNAS server, `Port Type` to `HTTP` and `Service Port` to `81`.

It's prefered to use https (even self-signed) when trying to use services outside the cluster using external service

>  hostname of your TrueNAS server
Super confusing, because it isn't the hostname of the truenas server (according to the official docs) you're refering to.

> You can set the `Host` and `HostName` values
Keep subjects seperate.
Also: External service itself does not contain these, it's "setup Ingress like the Ingress guide instructs".
Thats also why it need to be in a later guide aswell, because users in this stage are not supposed to know about ingress yet.

> Ensure your DNS points truenas.example.com to the ip address of your TrueNAS server.

This is not specific for this usecase and also our official advice is to use k8s-gateway.
It would also be one of the topics of a future quick-start guide (centered about DNS).


If you want to keep this as an example usecase (so not cut it into pieces), I suggest creating an "community examples" catagory (folder) and creating a seperate article in there. Including screenshots preferably.

Reply:
Thanks for the feedback. I think it would be nice to document this somewhere :)

Yes normally it's best to use HTTPS for external services, even self signed. But the TrueNAS web gui is not really an external service, right? Traefik and TrueNAS web gui are on the same box. So in this case I think it's better to use HTTP, no? Other apps behind Traefik use HTTP too.

I know nothing of k8s or k8s-gateway... just thought I should mention something about DNS since `truenas.example.com` won't magically work by specifying the `Host` and `HostName` values.

O.k. so instead of hostname it's better to say domain name?

I was planning to modify the text and put it under `Additional Documentation` in `16 - Setting up External-Services`. But you prefer that I make an article inside a "community examples" catagory (folder)?

Reply:
> Other apps behind Traefik use HTTP too.

That's inter-cluster traffic, this is not.
There is no downside to using https in this case and it's not inside the cluster, so it should be used.

> I know nothing of k8s or k8s-gateway... just thought I should mention something about DNS since truenas.example.com won't magically work by specifying the Host and HostName values.

The thing is: You are giving an example, not writhing a quick-start guide yourself.

> O.k. so instead of hostname it's better to say domain name?

It's best to refer to the already existing Ingress guide.

> I was planning to modify the text and put it under Additional Documentation in 16 - Setting up External-Services. But you prefer that I make an article inside a "community examples" catagory (folder)?

Yes, I would.
Because even under "Additional Documentation" in "16 - Setting up External-Services" it would become a oneliner "you can also do this with the SCALE UI", due to all the information that should be left out or is already explained in the guide itself.